metron-interface/metron-rest/src/test/java/org/apache/metron/rest/service/impl/HdfsServiceImplTest.java

```diff
@@ -65,6 +66,7 @@ public void listShouldListFiles() throws Exception {
         FileUtils.writeStringToFile(new File(testDir, "file2.txt"), "value2");
 
         List<String> paths = hdfsService.list(new Path(testDir));
+        Collections.sort(paths, String::compareTo);
```

Comment:
This is nitpicky, but why even specify String::compareTo? Collections.sort(paths) uses compareTo by default.